praw/models/reddit/live.py

```diff
@@ -110,7 +111,8 @@ def __call__(self):
 
         """
         url = API_PATH['live_contributors'].format(id=self.thread.id)
-        return self.thread._reddit.get(url)
+        temp = self.thread._reddit.get(url)
+        return temp if isinstance(temp, RedditorList) else temp[0]
```

Comment:
Do both these situations actually occur now? Or is it always a list? If the latter, I would prefer to only support that.